### PR TITLE
upload_images

### PR DESCRIPTION
**Pull request by bodyyyya**

### CODE REVIEW

**Comment by bodyyyya** [comment]
i added option to our ROR market, function to upload images for product

**Comment by sanyakosivchuk** [comment]
тапкі

**Comment by sanyakosivchuk** [comment]
Good job braza, keep it up!

**Review by sanyakosivchuk — APPROVED** [review]
Basic functional realization is pretty good, i personally think that we gotta make it more more complicated in future, and also we gotta to try js or css to make our site appearance
more user friendly, and better looking.

File: app/models/product.rb [hunk 1]
```diff
@@ -2,4 +2,5 @@ class Product < ApplicationRecord
     validates :name, presence:true
     validates :description, presence:true
     validates :price, presence:true
+    has_one_attached :image
```
Comment (Jedoku) [inline app/models/product.rb]: try has_many_attached

**Review by Jedoku — CHANGES_REQUESTED** [review]
Tell Active Storage which service to use by setting Rails.application.config.active_storage.service. Because each environment will likely use a different service, it is recommended to do this on a per-environment basis. To use the disk service from the previous example in the development environment, you would add the following to config/environments/development.rb:

# Store files locally.
config.active_storage.service = :local

**Comment by Jedoku** [comment]
https://edgeguides.rubyonrails.org/active_storage_overview.html

**Review by Jedoku — APPROVED** [review]

**Pull request merged**